graylog2-web-interface/src/views/pages/ShowDashboardInBigDisplayMode.test.jsx

```diff
@@ -0,0 +1,98 @@
+}));
+
+describe('ShowDashboardInBigDisplayMode', () => {
+  const mockLocation = {
+    query: {
+      interval: '30', refresh: '10',
+    },
+  };
+
+  afterEach(() => {
+    cleanup();
+    jest.clearAllMocks();
+  });
+
+  it('sets refresh interval correctly based on location query', async () => {
+    render(<ShowDashboardInBigDisplayMode route={{}}
+                                          params={{ viewId: mockView.id }}
+                                          location={mockLocation} />);
+    await wait(() => expect(RefreshActions.setInterval).toHaveBeenCalledTimes(1));
+    await wait(() => expect(RefreshActions.setInterval).toHaveBeenCalledWith(10000));
+  });
+
+  it('should enable refresh actions', async () => {
+    render(<ShowDashboardInBigDisplayMode route={{}}
+                                          params={{ viewId: mockView.id }}
+                                          location={mockLocation} />);
+    await wait(() => expect(RefreshActions.enable).toHaveBeenCalledTimes(1));
+  });
+
+  it('should set new refresh interval when location query refresh param changes', async () => {
+    const { rerender } = render(<ShowDashboardInBigDisplayMode route={{}}
+                                                               params={{ viewId: mockView.id }}
+                                                               location={mockLocation} />);
+    rerender(<ShowDashboardInBigDisplayMode route={{}}
+                                            params={{ viewId: mockView.id }}
+                                            location={{ query: { ...mockLocation.query, refresh: '20' } }} />);
+
+    await wait(() => expect(RefreshActions.setInterval).toHaveBeenCalledTimes(2));
+    await wait(() => expect(RefreshActions.setInterval).toHaveBeenCalledWith(20000));
+  });
+
+  it('should not change RefreshActions when query refresh param did not changes', async () => {
+    const { rerender } = render(<ShowDashboardInBigDisplayMode route={{}}
+                                                               params={{ viewId: mockView.id }}
+                                                               location={mockLocation} />);
+    rerender(<ShowDashboardInBigDisplayMode route={{}}
+                                            params={{ viewId: mockView.id }}
+                                            location={mockLocation} />);
+
+    await wait(() => expect(RefreshActions.setInterval).toHaveBeenCalledTimes(1));
+    await wait(() => expect(RefreshActions.enable).toHaveBeenCalledTimes(1));
+  });
+
+  // it('should display view title', async () => {
+  //   const { getByText } = render(<ShowDashboardInBigDisplayMode route={{}}
+  //                                                               params={{ viewId: mockView.id }}
+  //                                                               location={mockLocation} />);
+
+  //   await waitForElement(() => getByText('view title'));
+  // });
```

Comment:
Question: At first I was not aware of the `connect` mock we are using for several other tests. The above tests are working without the `connect` mock, but at some point I needed to add the `cleanup` method in `afterEach`.
Implementing the `cleanup` method without the `conntect` mock throws the following error: `TypeError: desub is not a function`

But with the `connect` mock, the view returned from `ViewStore.getInitialState` is no longer available in e.g. the `BigDisplayModeHeader`

My question is, how to deal with this scenario?